official/resnet/cifar10_main.py

```diff
@@ -134,37 +134,43 @@ def train_preprocess_fn(image, label):
   # Randomly flip the image horizontally.
   image = tf.image.random_flip_left_right(image)
 
+  return image
+
+
+def parse_and_preprocess(record, is_training):
```

Comment:
Seems weird that we have both `parse_record` and `parse_and_preprocess`. Can we just combine them into one function, `parse_and_preprocess_record`? I think it should still work fine if we pass that combined function to the tests.